conf/battle/drops.conf

```diff
@@ -153,3 +153,35 @@ rare_drop_announce: 0
 // If RENEWAL_DROP, Bubble Gum, or any other modifiers are active autoloot
 // will take them into account.
 autoloot_adjust: 0
+
+// Does autoloot work when a monster is killed by mercenary only?
+mercenary_autoloot: no
+
+// Is getting items from a mercenary disabled when their master's idle?
+// Set to no, or the amount of seconds (NOT milliseconds) that need to pass before considering
+// a character idle.
+// Characters in a chat/vending are always considered idle.
+// A character's idle status is reset upon item use/skill use/attack (auto attack counts too)/movement.
+// Their master will only receive items if 'mercenary_autoloot' is activated,
+// otherwise they will be dropped on the ground as usual.
+// NOTE: This option uses a special timer to track idle time, separated from the normal idle timer.
+mer_idle_no_share: no
+
+// How the server should measure the mercenary master's idle time? (for mercenary autoloot ONLY) (Note 3)
```

Comment:
The comment "for mercenary autoloot ONLY" is not totally true, if we offer a script command?